app/src/pages/home.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+import Logo from '../images/logolu.png'
+import Intereses from '../components/intereses'
+import { LogoutButton } from '../auth0/logout'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faBars } from '@fortawesome/free-solid-svg-icons'
+import { useAuth0 } from "@auth0/auth0-react";
+import '../styles/home.css'
+
+
+function Home() {
+
+    const [menu, setMenu] = useState(false);
+    const showMenu = () => setMenu(!menu);
+    const [button, setButton] = useState(false);
```

Comment:
Group all the hooks declaration, first `useState`, then `useAuth0` and then the other things